site24x7/websitemonitor.go

```diff
@@ -283,6 +288,7 @@ func resourceDataToWebsiteMonitor(d *schema.ResourceData, client site24x7.Client
 		UserGroupIDs:          userGroupIDs,
 		ActionIDs:             actionRefs,
 		UseNameServer:         d.Get("use_name_server").(bool),
+		UpStatusCodes:          d.Get("up_status_codes").(string),
```

Comment:
The indentation looks weird here, can you `go fmt` the file?

Reply:
Yes sure :+1: